descriptors/ModuleDescriptor-template.json

```diff
@@ -141,6 +157,11 @@
       "permissionName": "search.config.languages.item.delete",
       "displayName": "Search - removes a supported language by id",
       "description": "Removes a supported language by id"
+    },
+    {
+      "permissionName": "search.index.inventory.reindex.post",
+      "displayName": "Search - starts a reindex operation",
+      "description": "Starts a reindex operation"
```

Comment:
Not consistent. `Inventory` is not mentioned in display name and description.

Reply:
Thanks, updated.